portfolio-website/script.js

```diff
@@ -0,0 +1,8 @@
+
+const input = document.getElementById("searchinput")
+const inputForm = document.querySelector(".d-flex")
```

Comment:
instead of querying by class. I would give the form an id and query by elementId